pkgs/development/compilers/gmqcc/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  pkgs,
+  stdenv,
+  fetchgit,
+  ...
+}:
+
+stdenv.mkDerivation rec {
+    pname = "gmqcc";
+    version = "unstable-2021-07-09";
+
+    src = fetchgit {
+      url = "https://github.com/graphitemaster/gmqcc";
+      rev = "297eab9e5e2c9cc4f41201b68821593a5cf9a725";
+      sha256 = "1hl2qn7402ia03kjkblj4q4wfypxkil99sivsyk2vrnwwpdp4nzx";
+    };
+
+    installPhase = ''
+      mkdir -p $out/bin
+      install gmqcc $out/bin
+    '';
```

Comment:
```suggestion
    installPhase = ''
      runHook preInstall
      
      mkdir -p $out/bin
      install gmqcc $out/bin
      
      runHook postInstall
    '';
```

Reply:
I've added the hooks back again now, and I've set permissions as requested.